pkg/render/manager.go

```diff
@@ -475,13 +488,23 @@ func (c *managerComponent) volumeMountsForProxyManager() []corev1.VolumeMount {
 
 // managerEsProxyContainer returns the ES proxy container
 func (c *managerComponent) managerEsProxyContainer() corev1.Container {
+	var keyPath, certPath string
+	if c.cfg.TLSKeyPair != nil {
```

Comment:
nit: This can never be nil.

Reply:
True, unless we write a bug into the code somewhere I guess.

Reply:
I copied this check from other places that we access it. I suggest we keep the check but add a comment so nobody is confused.